nmpolicy/internal/lexer/lexer.go

```diff
@@ -145,7 +147,7 @@ func (l *lexer) lexIdentity() (*Token, error) {
 
 		if l.isEOF() || l.isSpace() {
 			return token, nil
-		} else if l.isDot() || l.isEqual() || l.isColon() || l.isPlus() {
+		} else if l.isDot() || l.isEqual() || l.isColon() || l.isPlus() || l.isPipe() {
```

Comment:
Regarding, I meant that if you have `some.path==5|another.path:="whatever" - iiuc,, tokenizing the number 5 will fail.
So `|| l.isPipe()` should be added to `lexNumber` (line 105) as well.